roles/oc_setup/README.md

```diff
@@ -19,7 +19,28 @@ Access to a valid kubeconfig file via an `KUBECONFIG` environment variable.
 export KUBECONFIG=<kubeconfig_path>
 ```
 
+# Users
+
+The following users are created with their respective role. More information about roles is found in [Default cluster roles](https://docs.openshift.com/container-platform/4.17/post_installation_configuration/preparing-for-users.html#default-roles_post-install-preparing-for-users)
+
+| Username      | Role          | Description
+| ------------- | ------------- | -----------
+| nondadmin     | None          | No role is associated to this user.
+| basic_user    | basic-user    | A user that can get basic information about projects and users.
+| admin         | admin         | A project manager. If used in a local binding, an admin has rights to view any resource in the project and modify any resource in the project except for quota.
+| cluster_admin | cluster-admin | A super-user that can perform any action in any project. When bound to a user with a local binding, they have full control over quota and every action on every resource in the project.
```

Comment:
I believe the table in the `Users` section was already included in https://github.com/redhatci/ansible-collection-redhatci-ocp/pull/461 isn't?

Reply:
:weary:  my bad, I put this in the wrong branch, that explains why I was sure I added this somewhere but later didn't find it in the other branch :sweat_smile:  Thanks for checking, I'll remove this